From 008e04aebf15ac634ce15f0d579d50e3e9809711 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Fri, 15 Jan 2016 15:29:46 +0100
Subject: [PATCH] When assigning session to active VT prefer graphical sessions

I case when we launch graphical session on top of text session on the
same VT we want to mark newly created graphical session as active.
---
 src/ck-seat.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ck-seat.c b/src/ck-seat.c
index fedd900..6844c06 100644
--- a/src/ck-seat.c
+++ b/src/ck-seat.c
@@ -459,9 +459,9 @@ find_session_for_display_device (CkSeat     *seat,
         GList     *sessions;
         CkSession *session;
 
-        sessions = find_sessions_for_display_device (seat, device);
+        sessions = find_sessions_for_x11_display_device (seat, device);
         if (sessions == NULL) {
-                sessions = find_sessions_for_x11_display_device (seat, device);
+                sessions = find_sessions_for_display_device (seat, device);
         }
 
         if (sessions == NULL) {
-- 
2.5.0

